src/Functions.Worker.Extensions.OpenAI/Search/ConnectionInfo.cs

```diff
@@ -14,15 +14,24 @@ public class ConnectionInfo
     /// <param name="connectionName">
     /// The name of an app setting or environment variable which contains a connection string value.
     /// </param>
+    /// <param name="databaseName">
+    /// The name of the database which has the collection.</param>
+    /// </param>
     /// <param name="collectionName">
     /// The name of the collection or table to search or store.</param>
     /// </param>
     /// <param name="credentials">
     /// Credentials for authenticating with the search provider.</param>
     /// </param>
-    public ConnectionInfo(string connectionName, string collectionName, string? credentials)
+    public ConnectionInfo(
```

Comment:
As discussed, please take any additional properties from host file for now, modifying this file will impact all other search providers too, like Kusto and AI Search

Also, this is causing the failures in build, please check it for errors

Reply:
I have moved it to the config file for now, but we should add it to the ConnectionInfo class.